examples/calculator_bridge.cc

```diff
@@ -20,7 +20,7 @@
 #include "examples/calculator.h"
 
 void CalculatorBridge::Init(napi_env env, napi_value exports) {
-  napi_status status;
+  napi_status __attribute__((unused)) status;
```

Comment:
IMHO, we shouldn't write compiler dependent codes if possible.
But your point is right. So, let's set napi_generic_failure as an initial value. WDYT?

FYI, you can see the examples in https://nodejs.org/api/n-api.html#n_api_working_with_javascript_properties.